### DISCUSSION TITLE
Updated from 3.5 to 3.7. How would I regenerate dashboard thumbnails?

### DISCUSSION THREAD

**Discussion by RyanTamulevicz** [mirumee/saleor-dashboard#2291] [Q&A] [answered]
I just did an upgrade from 3.5 to 3.7, how would I go about getting all of the thumbnails to regenerate for the dashboard? Currently it just has the person. The upgrade guide just states that the thumbnail changes happened now how to fix them.

**Comment by patrys** [comment, marked as answer]
@RyanTamulevicz There is no regeneration necessary, the thumbnails should work out of the box. Can you explain exactly what's happening?

**Reply by RyanTamulevicz** [reply]
They are becoming little people icons https://gyazo.com/912d28404e26f04a9d1c7ed0bc5804ba

**Comment by RyanTamulevicz** [comment]
@patrys it looks like it made new thumbnails automatically, and I had to make them public in my S3, my bad :) thanks!
